example/style.css

```diff
@@ -22,10 +22,14 @@ body {
   right: -30rem;
 }
 
-.js-accordion [aria-hidden=true] {
+.glossary div[aria-hidden=true] {
   display: none !important;
 }
 
+.glossary button {
```

Comment:
Is there a chance somebody would have to use an `input` or `a` for this making this selector not work?